Compiler warning with g++ 6

```
game/game.cpp: In function ‘bool game::tvmode(bool, bool)’:
game/game.cpp:676:34: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
                 case CS_WAITING: if((waitmode && (!player1->lastdeath || lastmillis-player1->lastdeath >= 500)) || (force && focus != player1 && followmode && followaim()))
                                  ^~
game/game.cpp:677:34: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
                     return true; break;
```

--- a/src/game/game.cpp
+++ b/src/game/game.cpp
@@ -673,8 +673,7 @@ namespace game
             else switch(player1->state)
             {
                 case CS_SPECTATOR: if(specmode || (force && focus != player1 && followmode && followaim())) return true; break;
-                case CS_WAITING: if((waitmode && (!player1->lastdeath || lastmillis-player1->lastdeath >= 500)) || (force && focus != player1 && followmode && followaim()))
-                    return true; break;
+                case CS_WAITING: if((waitmode && (!player1->lastdeath || lastmillis-player1->lastdeath >= 500)) || (force && focus != player1 && followmode && followaim())) return true; break;
                 default: break;
             }
         }

